src/seedu/addressbook/commands/Command.java

```diff
@@ -15,6 +15,7 @@
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
     private int targetIndex = -1;
+    protected boolean mutatesData;
```

Comment:
this variable is not used anywhere in your implementation. you may wish to omit this.